docs/azure_arc_jumpstart/azure_arc_vsphere/resource_bridge/powershell/_index.md

```diff
@@ -66,17 +66,17 @@ The following README will guide you on how to use the provided PowerShell script
   az ad sp show --id 'ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7' --query '{name:displayName,objectId:objectId}'
   ```
   
-    ![Screenshot of Azure resource provider for vSphere](./01.png)
+    ![Screenshot of Azure resource provider for vSphere](./img/01.png)
 
 ### vCenter Prerequisites
 
-- VMware vSphere vCenter Server v6.7.
+- VMware vSphere vCenter Server 8.
```

Comment:
should we mention 7 or 8 since both are supported ?

Reply:
true, fixed